.idea/.gitignore

```diff
@@ -0,0 +1,3 @@
+# Default ignored files
```

Comment:
According to https://stackoverflow.com/questions/17049416/what-is-the-idea-folder, looks like it's okay to add the .idea folder to our repo. https://intellij-support.jetbrains.com/hc/en-us/articles/206544839 confirms these are fine to add. 

Is this necessary to be on the repo though?